blackbox/docs/sql/ddl/alter_table.txt

```diff
@@ -135,6 +135,12 @@ Shard rerouting can help solve several problems:
     * **"Hot Shards"**: Most of your queries affect certain shards only. These
       shards lie on a node that has insufficient resources.
 
+This command takes these :ref:`Routing Allocation Settings <conf_routing>` into
+account. Once an allocation occurs CrateDB tries (by default) to re-balance
+shards to an even state. In this case it could be helpful to disable shard
```

Comment:
"In this case"? Maybe give an example when it is not good to have automatic rebalancing. (e.g. importing data?)